lib/service_sign_in/check-update-company-car-tax.cy.yaml

```diff
@@ -8,10 +8,10 @@ choose_sign_in:
       url: https://www.tax.service.gov.uk/gg/sign-in?continue=/paye/company-car/details&accountType=individual&origin=PTA-companycar
       hint_text: Bydd gennych Ddynodydd Defnyddiwr (ID) os ydych wedi cofrestru i wneud pethau fel cyflwyno'ch Ffurflen Dreth Hunanasesiad ar-lein.
     - text: Defnyddio GOV.UK Verify
-      url: https://www.tax.service.gov.uk/paye/company-car/start-verify
+      url: https://www.signin.service.gov.uk/initiate-journey/hmrc-company-car-tax?journey_hint=uk_idp_sign_in
```

Comment:
This URL doesn't work, is the simpleId correct?

Reply:
The simpleId is correct but we don't have the `headlessStartpage` defined for this service in config

Reply:
this page is not live anyway, the service has been taken down with verify. The headless redirects to GG login anyway